frontend/src/features/performance/PerformanceInitial.tsx

```diff
@@ -0,0 +1,137 @@
+  const proptype = useInitialStore((state) => state.proptype)
+  const propnumber = useInitialStore((state) => state.propnumber)
+  const nominalPower = useInitialStore((state) => state.nominalPower)
+  const flightAltitude = useInitialStore((state) => state.flightAltitude)
+  const startMass = useInitialStore((state) => state.startMass)
+  const fuelMass = useInitialStore((state) => state.fuelMass)
+  const fuelcons = useInitialStore((state) => state.fuelcons)
+  const vmax = useInitialStore((state) => state.vmax)
+  const wmax = useInitialStore((state) => state.wmax)
+  const area  = useInitialStore((state) => state.area)
+  const aspectRatio = useInitialStore((state) => state.aspectRatio)
+  const cx0 = useInitialStore((state) => state.cx0)
+  const czmax = useInitialStore((state) => state.czmax)
+  const setProptype = useInitialStore((state) => state.setProptype)
+  const setPropnumber = useInitialStore((state) => state.setPropnumber)
+  const setNominalPower = useInitialStore((state) => state.setNominalPower)
+  const setFlightAltitude = useInitialStore((state) => state.setFlightAltitude)
+  const setStartMass = useInitialStore((state) => state.setStartMass)
+  const setFuelMass = useInitialStore((state) => state.setFuelMass)
+  const setFuelcons = useInitialStore((state) => state.setFuelcons)
+  const setVmax = useInitialStore((state) => state.setVmax)
+  const setWmax = useInitialStore((state) => state.setWmax)
+  const setArea = useInitialStore((state) => state.setArea)
+  const setAspectRatio = useInitialStore((state) => state.setAspectRatio)
+  const setCx0 = useInitialStore((state) => state.setCx0)
+  const setCzmax = useInitialStore((state) => state.setCzmax)
+
+  const [calculatePower] = usePower();
+  const [maxPower, setMaxPower] = useState(calculatePower(0))
+
+  useEffect(() => {
+    setMaxPower(() => calculatePower(flightAltitude))
+  }, [flightAltitude, calculatePower])
+
+  return (
+    <div className="flex w-full p-4">
+      <div className="flex flex-col w-80 mr-8 space-y-1">
+        <InputSlider
+          value={propnumber}
+          setter={setPropnumber}
+          step={1}
+          min={1}
+          max={2}
+          label="Number of Engines"
+          unit=""
+        />
+        <InputNumber
+          value={nominalPower}
+          setter={setNominalPower}
+          step={0.1}
+          label="Nominal Power"
+          unit="kW"
+        />
+        <InputNumber
+          value={startMass}
+          setter={setStartMass}
+          step={0.01}
+          label="Start Mass"
+          unit="kg"
+        />
+        <InputNumber
+          value={fuelMass}
+          setter={setFuelMass}
+          step={1}
+          label="Mass of Fuel"
+          unit="kg"
+        />
+        <InputNumber
+          value={fuelcons}
+          setter={setFuelcons}
+          step={0.001}
+          label="SFC"
+          unit="kg/kWh"
+        />
+        <InputNumber
```

Comment:
This is already defined in Power Unit/Propeller as Cruise Speed (you can multiply it times 1.2 to get max speed)

Reply:
This value has to work as user's input until we manage to calculate it in proper way (reference: Flight Mechanics Project 6).